### ISSUE TITLE
Error shows as "Instance of 'minified:a2'" in pre-built app

### ISSUE THREAD

**Issue by DanTup** [flutter/devtools#179]
When running from a built version of the app, this is how the error appears when there's no connection:

![screenshot 2019-01-29 at 4 24 31 pm](https://user-images.githubusercontent.com/1078012/51923257-7f7d2600-23e2-11e9-858d-8e768d7c10b5.png)

**Comment by devoncarew** [comment]
It's likely that the exception object isn't interesting to show.

**Comment by DanTup** [comment]
I found this in `showError`:

```dart
// Only display the error object if it has a custom Dart toString.
if (message.startsWith('[object ')) {
  message = null;
}
```

I think it's aiming to hide useless messages, but I guess this is a different type of useless error. I opened https://github.com/flutter/devtools/pull/180 which fixes it, as well as the null reference error in `showError` when this code is triggered.

**Comment by DanTup** [comment]
(fixed in #180)
